gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -306,6 +319,13 @@ const NxDropdownPage = () =>
       This dropdown contains more complex JSX in its label.
     </GalleryExampleTile>
 
+    <GalleryExampleTile title="Custom label with NxThreatIndicator example"
+                        id="nx-dropdown-with-nx-threat-indicator-example"
+                        liveExample={NxDropdownWithNxThreatIndicatorExample}
+                        codeExamples={nxDropdownWithNxThreatIndicatorExampleCode}>
+      An example of a dropdown with <NxCode>NxThreatIndicator</NxCode>.
```

Comment:
Might want to call out that `nx-dropdown-icon` is _not_ used here and _is_ used in the example above.

Reply:
> Might want to call out that `nx-dropdown-icon` is _not_ used here and _is_ used in the example above.

We added it here: f7052b6057df0e5d4a37409ddf30b9e601d5be42